File: PHP Word Examples/SectionsApiDemo/updatePageSetup.php
<?php

use Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Api\SectionsApi;
use Spire\Cloud\Word\Sdk\Client\Model\LineNumber;
use Spire\Cloud\Word\Sdk\Client\Model\Margin;
use Spire\Cloud\Word\Sdk\Client\Model\PageSetup;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new SectionsApi($configuration);

$name = "PageSetup.docx";
$folder = 'input';
$storage = null;
$section_index = 0;
$page_setup = new PageSetup(array(
    'page_width' => 400,
    'page_height' => 500,
    'linenumber' => new LineNumber(array(
        'start_value' => 1,
        'step' => 2,
        'distance_from_text' => 20,
        'restart_mode' => LineNumber::RESTART_MODE_CONTINUOUS //RESTART_MODE_RESTART_SECTION, RESTART_MODE_RESTART_PAGE
    )),
    'page_margin' => new Margin(array(
        'top' => 20,
        'bottom' => 20,
        'left' => 20,
        'right' => 20
    )),
    'orientation' => PageSetup::ORIENTATION_LANDSCAPE //ORIENTATION_PORTRAIT
));
$password = null;
$dest_file_path = "Sections/UpdatePageSetup_1.docx";
$apiInstance->updatePageSetup($name, $section_index, $page_setup, $dest_file_path, $folder, $storage, $password);

File: Python Word Examples/SectionsApiDemo/updatePageSetup.py
import spirecloudword
from spirecloudword import LineNumber, PageSetup, Margin
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.SectionsApi(configuration)

file_name = 'PageSetup.docx'
section_index = 0
linenumber = LineNumber(start_value=1, step=2)
#supports modes:Continuous/RestartPage/RestartSection
linenumber._restart_mode = 'Continuous'
linenumber._distance_from_text = 20
page_setup = PageSetup()
page_setup._page_width = 400
page_setup._page_height = 500
page_setup._page_margin = Margin(20, 20, 20, 20)
#supports orientations:Landscape/Portrait
page_setup._orientation = 'Landscape'
page_setup._linenumber = linenumber
extra = {'folder': "input", 'dest_file_path': 'output/UpdatePageSetup.docx'}
result =api.update_page_setup(file_name, section_index, page_setup, **extra)